libr/include/r_arch.h

```diff
@@ -271,6 +252,13 @@ R_API void r_arch_op_fini(RAnalOp *op);
 R_API void r_arch_op_free(void *_op);
 #endif
 
+#if 1
+// Deprecate!
```

Comment:
```suggestion
// R2_590 Deprecate!
```
(or some other kind of TODO so we don't forget)